react/src/lib/components/DeckGLMap/DeckGLMap.tsx

```diff
@@ -72,6 +72,8 @@ export interface DeckGLMapProps {
      */
     getTooltip?: TooltipCallback;
     cameraPosition?: ViewStateType | undefined;
+
+    triggerResetOption?: Record<string, boolean>;
```

Comment:
The prop type need not to be Record<string, boolean>
We can make it consistent with other triggers like triggerHome.